packages/core/src/lib/light_push/light_push.ts

```diff
@@ -17,171 +15,132 @@ import { Uint8ArrayList } from "uint8arraylist";
 
 import { StreamManager } from "../stream_manager/index.js";
 
-import { PushRpc } from "./push_rpc.js";
-import { isRLNResponseError } from "./utils.js";
+import { CODECS, ProtocolHandler } from "./public.js";
 
 const log = new Logger("light-push");
 
-export const LightPushCodec = "/vac/waku/lightpush/2.0.0-beta1";
+export const LightPushCodecV2 = CODECS.v2;
+export const LightPushCodec = CODECS.v3;
+export const LightPushCodecs = [LightPushCodec, LightPushCodecV2];
 export { PushResponse };
 
-type PreparePushMessageResult = ThisOrThat<"query", PushRpc>;
-
 /**
  * Implements the [Waku v2 Light Push protocol](https://rfc.vac.dev/spec/19/).
  */
 export class LightPushCore {
   private readonly streamManager: StreamManager;
 
-  public readonly multicodec = LightPushCodec;
+  public readonly multicodec = CODECS.v2;
+  public readonly multicodecs = LightPushCodecs;
```

Comment:
`multicodes` are not used properly here on in LightPush SDK

they exist to satisfy this:

https://github.com/waku-org/js-waku/blob/103f21ef8532ed73f46ffcc2b899b84e7da55a94/packages/sdk/src/waku/waku.ts#L185

delete `multicodec` here and in `LightPush` SDK, keep only `multicodecs` and in `waku.ts` consume it